includes/plugins/class-woocommerce-memberships.php

```diff
@@ -117,6 +133,10 @@ function ( $list_id ) use ( $list_ids ) {
 	 */
 	public static function remove_user_from_list( $user_membership, $old_status, $new_status ) {
```

Comment:
Given that this method doesn't strictly remove a user from a list, I think it'd be good to change its name to something like `handle_membership_status_change`.

Reply:
Looking good: https://github.com/Automattic/newspack-newsletters/pull/1377/commits/1082a5a6cb27696bec2301523c5d6c5d2774d0cc